src/functionalTest/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterTest.java

```diff
@@ -53,11 +57,14 @@
 
     private final CompletableFuture<Void> voidCompletableFuture = CompletableFuture.completedFuture(null);
 
-    private static final String LETTER_JSON = "{"
-        + "\"template\": \"abc\","
-        + "\"values\": { \"a\": \"b\" },"
-        + "\"type\": \"typeA\""
-        + "}";
+    private static String LETTER_JSON;
+
+    @BeforeClass
+    public static void initializeLetter() throws IOException {
+        LETTER_JSON = new String(
+            readAllBytes(Paths.get("src/functionalTest/resources/letter.json"))
```

Comment:
would it not be better to use `getClass().getResource...` to grab file contents to string?